libmscore/scoreElement.h

```diff
@@ -399,7 +399,8 @@ class ScoreElement
 #undef CONVERT
 
     virtual bool isElement() const { return false; }   // overridden in element.h
-    bool isChordRest() const { return isRest() || isChord() || isRepeatMeasure() || isMMRest(); }
+    bool isRestFamily() const { return isRest() || isMMRest() || isMeasureRepeat(); }
```

Comment:
The reason I suggested "Type" here is that I was thinking of BarLineType that we used to check for a few related segments types in the same way.  But, that's a bit different, so maybe not important to mimic its naming.